scvi/module/_mrdeconv.py

```diff
@@ -43,8 +43,14 @@ class MRDeconv(BaseModuleClass):
         Mean parameter for each component in the empirical prior over the latent space
     var_vprior
         Diagonal variance parameter for each component in the empirical prior over the latent space
+    weight_vprior
+        Weight parameter for each component in the empirical prior over the latent space
     amortization
         which of the latent variables to amortize inference over (gamma, proportions, both or none)
+    l1_sparsity
+        Weighting in loss of L1 regularization of cell type proportions
+    beta_prior_weighting
```

Comment:
polish the docstring (ex.)
scalar parameter indicating the strength of the empirical variance penalty for the multiplicative offset in gene expression values (beta parameter)

Do the same for l1_sparsity

Reply:
call it beta_reg?

Reply:
also, indicate the values you think are good to try based on your experiments